### ISSUE TITLE
168 in 1 Contra Function not working

### ISSUE THREAD

**Issue by altlyser** [KMFDManic/NESC-SNESC-Modifications#180]
Hello. 168 in 1 NES game not working on hakchi CE 3.5.3.
I have a Km retroarch 177 Extreme, and command line:
--retroarch not work
--retroarch --core nestopia not work
--retroarch --core fceumm not work
https://www.youtube.com/watch?v=tqC90vkdhcQ
thanks

**Comment by KMFDManic** [comment]
This mapper does not appear to be currently supported, unfortunately.  Some additional mappers have been fixed up for my Next Release.  But, there is no guarantee, as of yet, this one will work.  I noticed that someone had before "changed the mapper" in at least one source for the game.  That version may have a chance of possibly working on FCEUmm or Nestopia.  I will try to give that changed mapper version a go later today.
